README.md

```diff
@@ -41,142 +23,74 @@ or if you use `yarn`
 yarn add did-jwt
 ```
 
-## API
-
-### Creating a JWT
-
-Use the `createJWT()` function
+## Example
 
+### 1. Create a did-JWT
```

Comment:
Woud be good to have a link to the api reference for the function described.
Same thing for the `2. Verify a did-JWT` section.

Reply:
Great suggestion!   We need to do this in many places, starting here :)

Fixed.